src/extensions/scratch3_vizblocks/index.js

```diff
@@ -136,6 +155,197 @@ class Scratch3VizBlocks {
         }
         return penState;
     }
+
+    getInfo () {
+        return {
+            id: 'vizblocks',
+            name: 'VizBlocks',
+            blockIconURI: blockIconURI,
+            blocks: [
+                {
+                    opcode: 'drawAxis',
+                    blockType: BlockType.COMMAND,
+                    text: formatMessage({
+                        id: 'vizblocks.drawAxis',
+                        default: 'draw axis',
+                        description: 'draw axis'
+                    })
+                },
+                {
+                    opcode: 'read',
+                    blockType: BlockType.COMMAND,
+                    text: formatMessage({
+                        id: 'vizblocks.read',
+                        default: 'read data x:[X] y:[Y]',
+                        description: 'read data'
+                    }),
+                    arguments: {
+                        X: {
+                            type: ArgumentType.NUMBER,
+                            defaultValue: 0
+                        },
+                        Y: {
+                            type: ArgumentType.NUMBER,
+                            defaultValue: 0
+                        }
+                    }
+                },
+                {
+                    opcode: 'drawLine',
+                    blockType: BlockType.COMMAND,
+                    text: formatMessage({
+                        id: 'vizblocks.drawLine',
+                        default: 'draw line',
+                        description: 'draw line'
+                    })
+                },
+                {
+                    opcode: 'clear',
+                    blockType: BlockType.COMMAND,
+                    text: formatMessage({
+                        id: 'vizblocks.clear',
+                        default: 'clear',
+                        description: 'clear canvas'
+                    })
+                }
+            ]
+        };
+    }
+
+    /**
+     * The pen "clear" block clears the pen layer's contents.
```

Comment:
Update comment:
`Clears the pen layer's contents.`

Reply:
comments updated